src/CharacterList.tsx

```diff
@@ -16,51 +14,31 @@ interface Vehicle {
 
 const ListItem = ({ person }: { person: Character }): JSX.Element => {
 
-  // const [vehicles, setVehicles] = useState<Vehicle[]>([])
-
-  // const getData = async (url: string) => {
-  //   const { data } = await axios.get(url);
-  //   return data;
-  // }
-
-  // useEffect(() => {
-  //   const vehicleData = person.vehicles.map(x => getData(x))
-  //   console.log(vehicleData)
-  //   Promise.all(vehicleData).then(x => setVehicles(x))
-  // }, [])
-
-  const query = (vehicleUrl: string) => {
-    return fetch(vehicleUrl).then(res => res.json())
-  }
-
-  const { isLoading, error, data } = useQuery(["getVehicle", vehicleUrl], () => query(vehicleUrl))
-
-  // console.log({ isLoading, data })
-
-  const queries = person.vehicles.map((v, i) => {
-    return {
-      queryKey: ["vehicle", i + 1],
-      queryFn: () => {
-        fetch(v).then(res => res.json())
+  const vehicleData = useQueries<Vehicle[]>(
```

Comment:
we were very close with that mapping but just not quite had the formatting correct. I also think we might have been using the docs for the latest version v4 but had v3 installed!